### DISCUSSION TITLE
Plate with centered circular hole.

### DISCUSSION THREAD

**Discussion by Leni-Yeo** [idaholab/moose#21373] [Q&A Modules: General] [answered]
Good afternoon, I am trying to create a plate with centered circular hole for uni-axial tension test. I was wondering if moose has a set up already that I may have missed or if anyone has a recommendation on how to set the mesh.  
Thank you!

**Comment by dschwen** [comment, marked as answer]
Yes, we have a way of doing that. Hang on, I'll find an example....

**Reply by dschwen** [reply]
It boils down to using `ConcentricCircleMeshGenerator` with `has_outer_square = true` and then using `BlockDeletionGenerator` to delete the inner circular block...

**Reply by Leni-Yeo** [reply]
Thank you!. I will try that.

**Reply by dschwen** [reply]
To get a 3D mesh you can add a `MeshExtruderGenerator`.

**Reply by Leni-Yeo** [reply]
It worked. Thank you!
